bluebrain/deployment/environments/applications_hpc.yaml

```diff
@@ -26,31 +26,29 @@ spack:
         - unit-test-translator
       projections:
         all: '{name}/{version}'
-        ^coreneuron+knl: '{name}-knl/{version}'
+        ^neuron+knl: '{name}-knl/{version}'
         +common: '{name}/{version}-commonmods'
         +plasticity: '{name}-plasticity/{version}'
         +ngv+metabolism: '{name}-multiscale/{version}'
   specs:
     - asciitoh5@1.0
-    - coreneuron%intel+tests
-    - coreneuron%intel+tests+nmodl+sympy
-    - coreneuron%nvhpc+tests+gpu+openmp
-    - coreneuron%nvhpc+tests+gpu+openmp+nmodl+sympy
-    - model-neocortex%intel
+    - neuron%intel+tests+coreneuron
+    - neuron%intel+tests+nmodl+sympy+coreneuron
+    - neuron%nvhpc+coreneuron+tests+gpu
+    - neuron%nvhpc+tests+gpu+openmp+coreneuron
+    - neuron%nvhpc+tests+gpu+openmp+nmodl+sympy+coreneuron
```

Comment:
Do we have any `neuron~coreneuron`? If `+coreneuron` is the project default, either that should come from the package itself or `packages.yaml`

Reply:
I agree!
In this PR I tried to do minimal changes i.e. replace coreneuron with neuron+coreneuron. My idea is to get `9.0a` deployed in production and then completely revamp neuron as well as all neurodamus* recipes. Currently it's just :spaghetti:!